### ISSUE TITLE
Support for Symfony 5.1.5

### ISSUE THREAD

**Issue by pibrom** [antishov/StofDoctrineExtensionsBundle#23]
Hello,
Symfony 5.1.5 has just been released today !
When I tried to install your PAckage, I encountered a issue : 
"Your requirements could not be resolved to an installable set of packages."

Is your package fully compatible with Symfony 5.1.5 Or do I need to install anything else before considering installign your package ?

Thanks for your help !

**Comment by numerogeek** [comment]
Hello !

Same problem here : 

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: remove doctrine/cache 1.10.2
    - Conclusion: don't install doctrine/cache 1.10.2
    - doctrine/common 2.2.1 conflicts with doctrine/cache[1.10.2].
    - doctrine/common 2.2.2 conflicts with doctrine/cache[1.10.2].
    - doctrine/common 2.2.3 conflicts with doctrine/cache[1.10.2].
    - doctrine/common 2.3.0 conflicts with doctrine/cache[1.10.2].
    - Installation request for doctrine/cache (locked at 1.10.2) -> satisfiable by doctrine/cache[1.10.2].
    - Installation request for antishov/doctrine-extensions-bundle 1.4.1 -> satisfiable by antishov/doctrine-extensions-bundle[v1.4.1].
    - Conclusion: don't install doctrine/common 3.0.2|install doctrine/common 2.2.1|install doctrine/common 2.2.2|install doctrine/common 2.2.3|install doctrine/common 2.3.0

```

And so on...

**Comment by numerogeek** [comment]
@pibrom
temporary fix : 
`composer remove doctrine/orm doctrine/doctrine-bundle doctrine/doctrine-migrations-bundle`
`composer require stof/doctrine-extensions-bundle`
`composer require  symfony/orm-pack
`

Doing this, you will install Doctrine 2 instead of 3.

**Comment by antishov** [comment]
Fixed in v1.4.3
Thanks
